schema/crdb/dbinit.sql

```diff
@@ -2899,14 +2847,67 @@ CREATE TABLE IF NOT EXISTS omicron.public.db_metadata (
     CHECK (singleton = true)
 );
 
+
+CREATE SEQUENCE IF NOT EXISTS omicron.public.ipv4_nat_version START 1 INCREMENT 1;
```

Comment:
This is a super minor nitpick, but we did some work to ensure that the order of declarations in `dbinit.sql` should **mostly** not matter.

Can we move this stuff up earlier? I find it a little odd to have the ordering of dbinit be:

- Most of the structures we use
- The db_metadata table declaration
- The tables that were added for NAT
- INSERT into db_metadata

(I kinda think db_metadata's declaration + INSERT should be right next to each other, at the end of the file, if opssible)